nextcord/application_command.py

```diff
@@ -1582,6 +1582,11 @@ def __init__(
 
     # Simple-ish getter + setter methods.
 
+    @property
+    def qualified_name(self) -> str:
+        """:class:`str`: Retrieves the fully qualified command name."""
```

Comment:
```suggestion
        """:class:`str`: Retrieves the fully qualified command name.

        .. versionadded:: 2.1
        """
```
sorry, forgor this docstring too